monai/bundle/workflows.py

```diff
@@ -316,6 +333,23 @@ def _set_property(self, name: str, property: dict, value: Any) -> None:
             self._is_initialized = False
             self.parser.ref_resolver.reset()
 
+    def add_property(self, name: str, required: str, config_id: str, desc: str | None = None):
```

Comment:
There is 1 more arg `config_id: str` in this subclass function.

Thanks.

Reply:
Thanks Nic. If so, could `config_id` be an optional arg, and then we can combine the logic into one function? Otherwise we will have two similar functions.